bluemira/equilibria/equilibrium.py

```diff
@@ -1225,7 +1226,7 @@ def fRBpol(self, psinorm):
         """
         Get f = R*Bt at specified values of normalised psi.
         """
-        return self._profiles.fRBpol(psinorm) * np.abs(self._profiles.scale)
+        return self._profiles.fRBpol(psinorm)
```

Comment:
yeah, I still need to have a better think about this. no headspace atm :/

Reply:
as things stand I do think it is necessary, but unfortunately because our tests are done with an old BLUEPRINT equilibrium, which isn't necessarily the right "truth" the private test is failing. I'd like to get #570 in before fixing these tests, as the relative tolerances to the truth will change when then gets pulled in too.